rts/motoko-rts/src/stabilization/deserialization.rs

```diff
@@ -92,24 +93,16 @@ impl Deserialization {
             |context, field_address| {
                 *field_address = translate(context, *field_address);
             },
-            |context, slice_start, array| {
-                const SLICE_INCREMENT: usize = 127;
```

Comment:
From previous PR, note that SLICE_INCREMENT is now 128 (not 127) in slice_array (I think).

Reply:
Thanks for mentioning. I think this is fine. Any value `>= TAG_ARRAY_SLICE_MIN` should be correct.